locales/de-DE.json

```diff
@@ -144,18 +201,58 @@
     "content_warning": "Schreib hier deine Warnung",
     "default_1": "Was geht dir gerade durch den Kopf?",
     "reply_to_account": "Antwort an {0}",
-    "replying": "Antworten"
+    "replying": "Antworten",
+    "the_thread": "der Thread"
+  },
+  "pwa": {
+    "dismiss": "Zurückweisen",
```

Comment:
this is a weird translation IMO, but i don't really have a better one. maybe just "ignorieren", or "entfernen" ?

Reply:
```suggestion
    "dismiss": "Ignorieren",
```

Reply:
"Verwerfen" is the more common frase for this.

Reply:
Keep it in mind when we touch de-DE again or open a quick mini PR if you like to